activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/controller/ModelController.java

```diff
@@ -149,6 +150,9 @@ public void updateModelContent(
         @PathVariable String modelId,
         @RequestPart(UPLOAD_FILE_PARAM_NAME) MultipartFile file
     ) throws IOException {
+        if (file.getSize() > 1024 * 1024 * 10) {
```

Comment:
please replace the number (1024*1024*10) with a property using @Value annotation

@Value("${activiti.modeling.maxModelFileSize:10485760}")
private long maxModelFileSize;